viewer/package.json

```diff
@@ -80,7 +80,7 @@
   },
   "devDependencies": {
     "@azure/msal-browser": "^3.13.0",
-    "@cognite/sdk": "^10.0.0",
+    "@cognite/sdk": "^10.1.3",
```

Comment:
Is this change necessary in this PR?

Reply:
Yes, we rely on newer annotation types defined in recent SDK versions